src/source/ObjcGenerator.scala

```diff
@@ -417,6 +420,44 @@ class ObjcGenerator(spec: Spec) extends Generator(spec) {
       }
       w.wl
 
+      w.wl("- (NSDictionary *)toDict")
+      w.braced {
+        w.wl("#define _djinni_hide_null_(_o_) ((_o_)?(_o_):([NSNull null]))")
```

Comment:
Is a macro really necessary? It's a code generator, no reason to avoid repetetive output.
If it absolutely has to be a macro add an`#undef` when it's no longer needed.